lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,59 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+      attr_reader :environment, :options, :name
```

Comment:
I'd prefer a integration test, you could have a fixture app that has db/migrations and apps/web/db/migrations and run migration with Ruby's `system` method then check to see if migration is applied correctly, please see my integration test on lotus/model branch for ideas.